core/src/components/datetime-button/datetime-button.tsx

```diff
@@ -206,6 +206,10 @@ export class DatetimeButton implements ComponentInterface {
      */
     const parsedDatetimes = parseDate(parsedValues.length > 0 ? parsedValues : [getToday()]);
 
+    if (!parsedDatetimes) {
```

Comment:
Just so I understand: `parseDate` has always been able to return `undefined`, we just never accounted for it, right?

Reply:
Right. When setting up the type signature overloads, I missed that `undefined` gets returned if the date wasn't parsed correctly; at the time I thought this was only for `undefined`/`null` being passed in.

Reply:
Awesome, thanks!